homeassistant/components/statistics/sensor.py

```diff
@@ -266,6 +279,25 @@ def _purge_old(self):
                 self.change = self.average_change = STATE_UNKNOWN
                 self.change_rate = STATE_UNKNOWN
 
+        # If max_age is set, ensure to update again after the defined interval.
+        next_to_purge_timestamp = self._next_to_purge_timestamp()
+        if next_to_purge_timestamp:
+            _LOGGER.debug(
+                "%s: scheduling update at %s.", self.entity_id, next_to_purge_timestamp
```

Comment:
We don't end logging messages with period.

Reply:
OK